### PR TITLE
Update: Bump webdriver version to 0.48

### LINKED ISSUE #29742: Upgrade the webdriver crate to version 0.48
Upgrading this crate will help us to gradually move to using only mio 0.8 and no longer rely on the forked version of mio in `Cargo.toml`. This requires updating all of the references in Cargo.toml and then fixing the build errors including adding Wheel event support to `WebDriverCommandMsg`.

### PR DESCRIPTION
**Pull request by mikayelgr**
These changes bump the `webdriver` crate from version 0.44 to 0.48. There are compilation issues which will need to be addressed accordingly.

---

- [X] `./mach build -d` does not report any errors
- [X] `./mach test-tidy` does not report any errors
- [X] These changes fix #29742 (GitHub issue number if applicable)

- [X] These changes may require some additional tests

### CODE REVIEW

**Comment by jdm** [comment]
You will need to update the list of ignored duplicate packages in https://github.com/servo/servo/blob/master/servo-tidy.toml#L27 to address the `./mach test-tidy` errors about new duplicate packages.

**Comment by michaelgrigoryan25** [comment]
@jdm is that related to my PR though? I honestly don't know which packages should be included in that list.

**Comment by michaelgrigoryan25** [comment]
Let's see whether ba07eab022310f4667aac1cd8390dd22f420a914 will fix that issue.

File: servo-tidy.toml [hunk 1]
```diff
@@ -65,14 +63,14 @@ packages = [
...
-  "hyper",
+  # "hyper",
   "bytes",
-  "pin-project",
+  # "pin-project",
   "pin-project-lite",
-  "pin-project-internal",
-  "socket2",
+  # "pin-project-internal",
+  # "socket2",
```
Comment (jdm) [inline servo-tidy.toml:73]: You can delete all of the commented out entries :)

**Comment by michaelgrigoryan25** [comment]
Yeah, that is expected. Currently, there is no implementation for ActionType::Wheel.

**Pull request merged**